src/components/common/PeopleAndRoles/CurrentDirectors.vue

```diff
@@ -131,7 +130,7 @@ export default class CurrentDirectors extends Mixins(CommonMixin) {
     vertical-align: text-top;
   }
 
-  td:not(:first-child){
+  .director-detail {
```

Comment:
No need to look for "all but the first td" since the spans we want to style already have this classname (and the first one, ie line 40, doesn't).